automation/constants.go

```diff
@@ -0,0 +1,9 @@
+package automation
+
+const (
+	DATABASE_PATH   = "./config/database.yaml"
+	DB_SECRET_PATH  = "./config/db-secret.yaml"
+	NDB_SECRET_PATH = "./config/ndb-secret.yaml"
+	APP_POD_PATH    = "./config/pod.yaml"
+	APP_SVC_PATH    = "./config/service.yaml"
```

Comment:
As discussed previously with @shenoypritika, these variables will change for different tests and might not have same values. Would it make sense to get rid of this file and define these constants within the specific test folder instead?